Sources/MountebankSwift/Models/Recreatable.swift

```diff
@@ -0,0 +1,176 @@
+import Foundation
+
+// MARK: - Protocol
+
+public protocol Recreatable {
```

Comment:
I wanted to make this `internal` or `package` but I could't get it compiling with `func swiftString(depth: Int) -> String` being a `func`. This did work when it still was `var swiftString: String { get }` btw, weirdly enough